EIPS/eip-5548.md

```diff
@@ -0,0 +1,126 @@
+
+At the time of writing, several intermediary platforms have chosen to disregard the widely agreed-upon convention of remitting resale royalties to creators. It is expected that the extension proposed in this EIP will frequently be paired with [EIP-2981](./eip-2981.md), with EIP-2981 acting as a way of signalling standardized royalty information, and this EIP acting as a method of denying non-compliant intermediary platforms.
+
+
+## Motivation
+
+The promise of creator empowerment has driven adoption of the NFT standard (and of the Ethereum blockchain in general) for artists and creators. The opportunity to cut out intermediaries, or "middlemen," is a central tenet not just of NFT culture, but of crypto culture in general. Nonetheless, intermediaries persist. In many cases, these intermediaries serve an important function, creating smoother experiences for end users, or presenting NFT metadata in easily consumable formats. 
+
+However, as intermediary platforms willfully circumvent social codes established by NFT creators, it is within the spirit of crypto to provide technological tools to deny allowances to those intermediaries.
+
+It should be noted that this extension will not (nor does it aim to) stamp out any nefarious behaviour (such as royalty-dodging) from token holders. The position of this EIP is that any restriction to owner's ability to `Transfer` their NFTs would be overreaching. And of course, denyListed platforms could deploy new smart contracts, placing creators in the position of playing "whack-a-mole" to add new deployments to the denyList. 
+
+Nonetheless, there is benefit to creating some punitive friction for bad actors, and to implementing an EIP-721 extension which signals creator's intention to exert control over the monetization of their creations.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+Our denyList is a single new mapping of operator addresses, paired with true/false values. An event is emitted when an address is added to the denyList.
+
+```solidity
+
+mapping (address => bool) public denyList;
+event OperatorDenied(address badOperator);
+
+```
+
+Addresses may be added to the denyList.
+
+
+
+```solidity
+
+function updateDenyList(address badOperator, bool status) public onlyOwner {
+       
+        require(
+            _msgSender() == owner ,
+            "ERC721: approve caller is not token owner nor approved for all"
+        );
+
+        _addToDenyList(badOperator, status);
+    }
+
+function _updateDenyList(address badOperator, bool status) internal virtual {
+        denyList[badOperator] = status;
+        emit OperatorDenied(badOperator);
```

Comment:
use the added boolean mentioned on line 44 comment unless you want to only deny

Reply:
Also added in most recent commit.